src/documentation/install.page

```diff
@@ -8,32 +8,32 @@ sort_info: 2
 lang: en
 ---
 
-# Install Uniqush System
+# Installing Uniqush and its Dependencies
 
-## Install uniqush-push
+## Installing uniqush-push
 
-uniqush-push now is the only component in the Uniqush System.
+uniqush-push now is the only component in Uniqush.
 
 It is higly recommanded to install from **binary** package on **64-bit** system.
 
 ### Install Redis
 
-The only dependency of uniqush-push is [redis], which is a key-value database.
+The only dependency of uniqush-push is [Redis], which is a key-value database.
 
-- On debian-based system (including [debian], [ubuntu], [mint], etc.), run:
+- On Debian-based systems (including [Debian], [Ubuntu], [Mint], etc.), run:
     sudo apt-get install redis-server
```

Comment:
Nest in backticks for monospace text on the same line. 

Add a newline above or use `~~~` to create a block of monospace text. Not completely sure which would work.